city_scrapers/spiders/cook_emergency_telephone.py

```diff
@@ -0,0 +1,135 @@
+import re
+from datetime import datetime, timedelta
+from io import BytesIO
+
+import scrapy
+from city_scrapers_core.constants import FORUM
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from PyPDF2 import PdfFileReader
+
+
+class CookEmergencyTelephoneSpider(CityScrapersSpider):
+    name = "cook_emergency_telephone"
+    agency = "Cook County Emergency Telephone System Board"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.cookcounty911.com/"]
+    location = {
+        "name": "Conference Room",
+        "address": "1401 S. Maybrook Drive, Maywood, IL 60153",
+    }
+
+    def __init__(self, *args, **kwargs):
+        self.meeting_starts = []
+        self.docs_link = ""
+        self.agenda_link = ""
+        super().__init__(*args, **kwargs)
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        schedule_link = ""
+        for link in response.css(".sliding_box a"):
+            link_text = link.css("*::text").get()
+
+            schedule_link = self.start_urls[0] + '/wp-content/uploads/pdfs/'
+
+            if "Minutes" in link_text:
+                self.docs_link = link.attrib["href"]
+            elif "Agenda" in link_text:
+                self.agenda_link = link.attrib["href"]
+
+        if schedule_link and self.docs_link:
+            yield scrapy.Request(
```

Comment:
Is this request necessary? It looks like `_parse_schedule` could request the page ending in "schedule.pdf" call `_parse_schedule_pdf` directly and then yield from `_parse_documents` instead of making a separate call